src/Routing/Filter/AssetFilter.php

```diff
@@ -118,8 +117,7 @@ protected function _deliverAsset(Request $request, Response $response, $assetFil
 			$response->header('Content-Length', filesize($assetFile));
 		}
 		$response->cache(filemtime($assetFile));
-		$response->send();
-		ob_clean();
+		$response->sendHeaders();
 		readfile($assetFile);
 		if ($compressionEnabled) {
 			ob_end_flush();
```

Comment:
Why me? :)

I don't have any problem with zlib.out_compression enabled or disabled in php.ini